source/includes/_events.md

```diff
@@ -1,5 +1,22 @@
 # Events
 
+Every call to action is represented in the API as an event.  Events have related resources such as a contact or a location.
```

Comment:
This isn't actually true, I think? Events are a category of calls to action, but phone calls are not events, and are still calls to action.